src/CNGLOB.H

```diff
@@ -84,6 +84,15 @@ typedef long long LLI;		// signed 64 bit integer
 typedef unsigned long long ULLI;
 //typedef int BOOL;	if needed: // 16 or 32 bit Boolean, matches windows.h.
 
+/*---------------------------- Remove DLL Support ---------------------------*/
+#if USE_XMODULE
+#define SUPPORT_XMODULE
+#pragma message("CNGLOG: SUPPORT_XMODULE is defined")
+#else
+#undef SUPPORT_XMODULE
+#pragma message("CNGLOG: SUPPORT_XMODULE is undefined")
+#endif // USE_XMODULE
```

Comment:
Can you think of a way to handle this where you don't need to define use both `USE_XMODULE` and `SUPPORT_XMODULE`?

Also, we don't need the `#pragma` messages. If the user chooses to build with XMODULE support it will be a fairly conscious decision and they don't need to be reminded of it.